### PR TITLE
Masking password value from request body

### PR DESCRIPTION
**Pull request by RifanNurmuhammad**

### CODE REVIEW

**Comment by bmd-antoniush** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/Golib-PR/9/

File: tracer/http_middleware.go [hunk 1]
```diff
@@ -46,3 +48,35 @@ func Middleware(h http.Handler) http.Handler {
 		h.ServeHTTP(w, req.WithContext(ctx))
 	})
 }
+
+// MaskPassword for mask password string
+func MaskPassword(s string) string {
```
Comment (hardyantz) [inline tracer/http_middleware.go]: baiknya simpen di file helper.go aja func nya.

**Comment by bmd-antoniush** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/Golib-PR/10/

**Comment by bmd-antoniush** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://build.bhinnekalocal.com:8080//job/Golib-PR/11/

**Pull request merged**